tests/unit/test_releaser.py

```diff
@@ -34,6 +36,15 @@ def balena_release():
     }
 
 
+def make_device(**kwargs):
+    """fixture factory for mocking a device
+    for current tests we only need the fields below and we can add more as needed
+    """
+    device_data = {"id": 5883496, "is_running__release": {"__id": 1234}}
+    device_data.update(kwargs)
+    return device_data
+
```

Comment:
I find it quite useful to generate test data  with factories for unit tests:  This way one has less places to update when the underlying model changes

Reply:
Agreed. The lack of this caused a timesink with schema updates in tmh-sch before, but I believe now we tried/try to build the testcases there similarly from one place to simplify that.